src/main/java/io/stargate/sgv2/jsonapi/service/resolver/DropTableCommandResolver.java

```diff
@@ -18,6 +27,24 @@ public Class<DropTableCommand> getCommandClass() {
   @Override
   public Operation resolveKeyspaceCommand(
       CommandContext<KeyspaceSchemaObject> ctx, DropTableCommand command) {
-    return new DropTableOperation(ctx, command.name());
+    final DropTableCommand.Options options = command.options();
+    final boolean ifExists = (options != null) && options.ifExists();
+    final SchemaAttempt.SchemaRetryPolicy schemaRetryPolicy =
```

Comment:
This code is duplicated, we should create one way to create the SchemaRetryPolicy from config. 

such as a static factory on the class that takes a OperationsConfig instance and gets the retry count and delay millis from it.

Also, this is different to the way the SchemaRetry is build for the CreateTableAttempt we need one way of doing things.